homeserver.yaml

```diff
@@ -1121,7 +1121,7 @@ form_secret: "loG32N*yCgr;Pp.Ih1l_fuw#G:0ii@D8J0cuPbed#-o25:sKLX"
 
 # Path to the signing key to sign messages with
 #
-signing_key_path: "/data/matrix.ocf.berkeley.edu.signing.key"
+signing_key_path: "/data/ocf.berkeley.edu.signing.key"
```

Comment:
Not sure if changing this might break

Reply:
I think we want to break it, switching the homeserver name requires a key + database reset or other servers will get confused.